### PR TITLE
New builder weapon-station

### PR DESCRIPTION
**Pull request by beccabunny**
**Content (Artwork / Missions / Jobs)**

## Summary
This PR adds a new, deactivated Builder station in Fearis, one of the two systems that have a stable wormhole in them. The description and appearance is the one of a central, small crew section with three massive guns around it in a triangular frame. In the files I also added a sprite of the repaired version of the station, for possible future usage. Similarly, a new, unused scene of the station firing is added to the files.
I also used a new landing sprite for it, taken from unsplash.
Screenshot (25)
*The station in game, orbiting a gas giant.*
Screenshot (24)
*The landed view.*
station4b firing@2x
*The scene representing the station firing.*

## PR Checklist
 - [x] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified
 - [x] I uploaded the necessary image, blend, and texture assets here: [weapon station.zip](https://github.com/endless-sky/endless-sky/files/5011034/weapon.station.zip)
 - [x] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: https://github.com/endless-sky/endless-sky-high-dpi/pull/133

### CODE REVIEW

File: copyright [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+﻿Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
```
Comment (MCOfficer) [inline copyright:1]: unintended whitespace change?

File: data/map.txt [hunk 2]
```diff
@@ -26871,6 +26871,12 @@ planet "Leviathan Station"
 		threshold 3500
 		fleet "Small Deep Security" 27
 
+planet "Letis'sei"
+	attributes ka'het
+	landscape land/loc3
+	description `Letis'sei was disabled long ago, but it seems the destruction it suffered was relatively small: with only a tiny fraction of the station designed to be inhabited, it didn't take much for the Ka'het to blow the crew quarters up. Today, all the air once inside the station is gone, and few of the original rooms can still be recognised.`
```
Comment (MCOfficer) [inline data/map.txt:26877]:
```suggestion
	description `Letis'sei was disabled long ago, but it seems the destruction it suffered was relatively small: with only a tiny fraction of the station designed to be inhabited, it didn't take much for the Ka'het to eliminate the crew. Today, all the air once inside the station is gone, and few of the original rooms can still be recognised.`
```

File: data/map.txt [hunk 3]
```diff
@@ -26871,6 +26871,12 @@ planet "Leviathan Station"
...
+	description `	Outside, three impressive, cannon-like structures slowly rotate around the living section. Each of them is more than twice as big as the crew quarters, and despite their age they still seem to be mostly intact.`
```
Comment (MCOfficer) [inline data/map.txt:26878]: twice as large*? native speakers, please confirm.

Comment (Anarchist2) [reply data/map.txt:26878]: Twice as big works fine ~~but that `is` needs to be an `are`~~

Comment (W1zrad) [reply data/map.txt:26878]: `Big` is fine but `large` might also sound more professional.

**Comment by Darcman99** [comment]
> What is this monstrosity?! awesome.
> 
> Not sure how our stance on unused assets is, maybe they should be moved to whatever future PR uses them.

There are plenty of unused scenes.

File: copyright (hunk 1 above)
Comment (beccabunny) [reply copyright:1]: What did change though?

File: data/map.txt (hunk 3 above)
Comment (beccabunny) [reply data/map.txt:26878]: Fixed

File: copyright (hunk 1 above)
Comment (oo13) [reply copyright:1]: You inserted a byte order mark (BOM) at the beginning of the file.
Please confirm your text editor settings.

Comment (beccabunny) [reply copyright]: Fixed, thank you

**Pull request merged**